scripts/remind_draft_preregistrations.py

```diff
@@ -47,9 +47,20 @@ def find_neglected_prereg_within_reminder_limit():
 
     already_queued = [entry['draft_id'] for entry in queue_data]
 
+    ## If project has been deleted since last time the script ran remove reminder email from queue.
+    drafts_for_deleted_projects = list(DraftRegistration.objects
```

Comment:
Minor: You can save a query by making this a subquery:

````python
QueuedMail.objects.filter(..., data__draft_id__in=Subquery(drafts_for_deleted_projects))
```

Reply:
Hm, apparently this doesn't work as expected because `draft_id` is a key in a JSONB field. So this is fine as is.